### PR TITLE
AWS GovCloud

### PR DESCRIPTION
**Pull request by resmaa**
**Release 3.1 Feature**

AWS Gov Cloud

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1050#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | d2e3f54b1f8e4ade5f107c8ad8acea67ef967cb2 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1050/librarium_build/1604504000533630976) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:966]: @resmaa this is a how to so we need to follow the How-to [layout guide](https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1774944263/How-to+Layout). This means we need to add prerequisites, the enablement steps, and a validation section.

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:968]:
```suggestion
Palette supports integration with [AWS GovCloud (US)](https://aws.amazon.com/govcloud-us/?whats-new-ess.sort-by=item.additionalFields.postDateTime&whats-new-ess.sort-order=desc). Using Palette you can deploy Kubernetes clusters to your AWS GovCloud account. To get started with AWS GovClud and Palette, use the following steps.
```
The current paragraph form uses awkward language and is confusing to read. Keep things simple.

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:970]:
```suggestion
```
We can remove this sentence.

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:972]:
```suggestion
1. In the AWS console, create the following four policies. You can find the IAM policy JSON for each policy in the [AWS Cloud Account Permissions](/clusters/public-cloud/aws#awscloudaccountpermissions) section. 
 - Controller Policy
 - Control Plane Policy
 -  Nodes Policy
 - Deployment Policy
 
 Ensure these policies are applied to the IAM User or IAM Role that Palette will use.
```
This helps clarify what is happening

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:975]:
```suggestion
2. Log in to Palette as a Tenant Admin.

```

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:978]:
```suggestion
3. Go to **Tenant Settings** > **Cloud Accounts** and click  on **+Add AWS Account**.

```

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:981]:
```suggestion
4. In the cloud account creation wizard, provide the following information:

```

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:987]:
```suggestion
       * AWS access key
       * AWS secret access key

```

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:990]:
```suggestion
5. Click on the **Validate** button to validate the credentials.  Once the credentials are validated, a toggle button appears as **Add IAM Policies**. 

 6. Toggle the button to add IAM policies. Add the required IAM policies you previously created. 
```
Step five is packed with multiple actions, break out each step into its own bullet.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Resma, we need to follow the style guide for how-to's. This page is also way too busy so we need to breakout the information in the future.

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:906]:
```suggestion
   * **Account Name:** Custom name for the cloud account.

```

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:968]: Replace first sentence with a Prerequisite section that lists 'AWS US Gov Cloud Account registered to the Palette Platform' as the prerequisite.

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:975]:
```suggestion
2. Log in to Palette as Tenant Admin.

```

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:984]:
```suggestion
   * **Partition:** Choose **AWS Gov Cloud** from the drop-down menu.

```

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (ritawatson) [inline content/docs/04-clusters/01-public-cloud/01-aws.md:982]:
```suggestion
   * **Account Name:** Custom name for the cloud account.

```

**Review by ritawatson — APPROVED** [review]
LGTM with some comments.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1050#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1050#pullrequestreview-1223279029" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | baac9d40965d6abf2a623d465dc5e9f702ad813e | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1050/librarium_build/1605691474987454464) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/04-clusters/01-public-cloud/01-aws.md
Comment (ritawatson) [reply content/docs/04-clusters/01-public-cloud/01-aws.md]: step 5 is a series of run-on sentences that need to be broken up. Some of these should probably be separate steps. (suggestion above)

Correct form is: drop-down menu.

**Comment by ritawatson** [comment]
Suggestion for line 914:

5. Click the **Validate** button. When the credentials are validated, the **Add IAM Policies** toggle displays. 
6. Toggle **Add IAM Policies** to *yes*. A  drop-down menu displays that lists attached policies.
7.  Choose a policy from the drop-down menu to attach it to the cloud account you created.

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 60e99b3aabe711f495954f5658ee0acd60336140 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1050/librarium_build/1605920991182065664) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 8a9d35dea21117375e6f5e1175f8388a1757977a | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1050/librarium_build/1605925143572385792) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1050#pullrequestreview-1227836165" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1050#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1050#pullrequestreview-1223279029" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
